### ISSUE TITLE
fontFile.replace is not a function [SDK 34]

### ISSUE THREAD

**Issue by bruno-edo** [expo/expo#5267]
### Environment
```
Expo CLI 3.0.6 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 3.0.2 - /usr/local/bin/fish
    Binaries:
      Node: 12.6.0 - /usr/local/bin/node
      Yarn: 1.17.3 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
    IDEs:
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^34.0.0 => 34.0.4 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-34.0.1.tar.gz => 0.59.8 
      react-navigation: ^3.11.1 => 3.11.1 
```

Targets: iOS (iPads).

### Steps to Reproduce

- Try to load custom icon font from IcoMoon.

### Expected Behavior

Icon fonts should be loaded and a ready for use as a component.

### Actual Behavior

![Error message](https://i.imgur.com/AKMs1B7.png)

### Reproducible Demo

Here's an [Expo Snack](https://snack.expo.io/ryR3Oxj7B) reproducing the error.

Observation: on my example I didn't use `Font.loadAsync` before trying to create the icon, but in my actual project I did and the results were the same.

**Comment by LeoMoreira1** [comment]
same issue here, but sdk 33

**Comment by cruzach** [comment]
As a workaround, take a look at the changes made in [104](https://github.com/expo/vector-icons/pull/104). Tested it locally and was able to render IcoMoon icons

**Comment by LeoMoreira1** [comment]
Nice! i don't get it exactly but if it works, it's good. I was looking at this functions in node_modules in my project folder and i thought that the problem was in createIconSet, when it modifies assetExpoId.

When it have be merged, i update my project's packages and give you a feedback

**Comment by cruzach** [comment]
Merged, published in v10.0.4
